app-frontend/src/app/services/scenes/datasource.service.js

```diff
@@ -36,7 +36,7 @@ export default (app) => {
                                     bands: _.map(
                                         payload.bands,
                                         band => {
-                                            if (band.wavelength) {
+                                            if (_.isString(band.wavelength)) {
```

Comment:
When would `band.wavelength` not be a string but also exist?

Reply:
In an API response, where we return an array

Reply:
I guess maybe `transformResponse` on `GET` might be the cleaner solution here

Reply:
No I changed my mind, that's not better -- because we've said in other places that we should reasonably expect to be able to POST back what we get from the API, and if I transform the array to a string, that won't be the case